src/frontend/screens/Settings/MapManagement/BackgroundMaps.tsx

```diff
@@ -154,11 +153,6 @@ export function BackgroundMapsScreen() {
                       uri: asset.uri,
                     },
                     {
-                      onError: () => {
-                        FileSystem.deleteAsync(asset.uri, {
-                          idempotent: true,
-                        }).catch(noop);
-                      },
```

Comment:
the asset here used to refer to the one in the cache directory, but given that we're not doing that in this specific case, there's no longer a need for this attempted deletion (i.e. we don't want to attempt to delete the original file, which I think fails anyways)